packages/react-components/src/components/VideoTile.tsx

```diff
@@ -151,7 +163,47 @@ const DefaultPlaceholder = (props: CustomAvatarOptions): JSX.Element => {
   );
 };
 
+/* @conditional-compile-remove(pinned-participants) */
+/**
+ * @internal
+ */
+export type VideoTileMenuItem = {
+  key: string;
+  ariaLabel?: string;
+  text: string;
+  onClick: () => void;
+  icon: IIconProps;
```

Comment:
Do you want to change `icon` to `iconProps` so that we don't need the helper function for mapping these props to contextual menu props?

Reply:
done